# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/PCookieStore.ipdl
# Commit: 7e736b5732c1
# Full Hash: 7e736b5732c17f105f4692be0794568e23f88532
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-12 09:23:07
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 10 - CookieStore API - CHIP implementation shared between Document and CookieStore, r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D220288
# ==============================================================================

diff -r 8620daea37ad -r 7e736b5732c1 dom/cookiestore/PCookieStore.ipdl
--- a/dom/cookiestore/PCookieStore.ipdl	Wed Sep 11 16:26:57 2024 +0000
+++ b/dom/cookiestore/PCookieStore.ipdl	Wed Sep 11 16:26:58 2024 +0000
@@ -32,6 +32,10 @@
 parent:
   async GetRequest(nsString domain,
                    OriginAttributes attrs,
+                   OriginAttributes? partitionedAttrs,
+                   bool thirdPartyContext,
+                   bool partitionForeign,
+                   bool usingStorageAccess,
                    bool matchName,
                    nsString name,
                    nsCString path,
@@ -39,6 +43,9 @@
 
   async SetRequest(nsString domain,
                    OriginAttributes attrs,
+                   bool thirdPartyContext,
+                   bool partitionForeign,
+                   bool usingStorageAccess,
                    nsString name,
                    nsString value,
                    bool session,
@@ -50,6 +57,9 @@
 
   async DeleteRequest(nsString domain,
                       OriginAttributes attrs,
+                      bool thirdPartyContext,
+                      bool partitionForeign,
+                      bool usingStorageAccess,
                       nsString name,
                       nsString path,
                       bool partitioned,